<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmarActControl</class>
 <widget class="QMainWindow" name="SmarActControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmarAct Device Controller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="devices">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>421</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>25</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Device:</string>
    </property>
   </widget>
   <widget class="QWidget" name="position" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>851</width>
      <height>216</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="close_button">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>340</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QAttribute" name="state" native="true">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QAttribute" name="is_cal" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>59</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QAttribute" name="is_ref" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_calibrate">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibrate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_reference">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reference</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>64</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Is calibrated</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>65</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Is referenced</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_reset">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>60</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Write Value</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_stop">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;</string>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAttribute</class>
   <extends>QWidget</extends>
   <header>PyQTango</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
